### PR TITLE
Added Hypervisor Manager to Collection

### PR DESCRIPTION
**Pull request by nabhajit-ray**
### Description
[Added Hypervisor Manager to Collection]

### Issues Resolved
[List any issues this PR will resolve. e.g., Fixes #01]

### Check List
- [ ] New functionality includes sanity testing.
  - [x] All sanity tests pass. (`$ ansible-test sanity`).
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).

### CODE REVIEW

File: plugins/modules/oneview_hypervisor_manager.py [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+DOCUMENTATION = '''
+---
+module: oneview_hypervisor_manager
+short_description: Manage OneView Hypervisor Managers resources.
+description:
+    - Provides an interface to manage Hypervisor Managers resources. Can create, update, and delete.
+version_added: "2.4.0"
+requirements:
+    - "python >= 3.4.2"
+    - "hpeOneView >= 5.2.0"
```
Comment (VenkateshRavula) [inline plugins/modules/oneview_hypervisor_manager.py:33]: change hpeOneView version to 5.4.0

File: plugins/modules/oneview_hypervisor_manager_facts.py [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+DOCUMENTATION = '''
+---
+module: oneview_hypervisor_manager_facts
+short_description: Retrieve the facts about one or more of the OneView Hypervisor Managers
+description:
+    - Retrieve the facts about one or more of the Hypervisor Managers from OneView.
+version_added: "2.4.0"
+requirements:
+    - "python >= 3.4.2"
+    - hpeOneView >= 5.2.0
```
Comment (VenkateshRavula) [inline plugins/modules/oneview_hypervisor_manager_facts.py:34]: change module version to 5.4.0

File: roles/oneview_hypervisor_manager/README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+## oneview_hypervisor_manager
+Manage OneView Hypervisor Manager resources.
+
+#### Synopsis
+ Provides an interface to manage Hypervisor Manager resources. Can create, update, or delete.
+
+#### Requirements (on the host that executes the module)
+  * hpeOneView >= 5.1.0
```
Comment (VenkateshRavula) [inline roles/oneview_hypervisor_manager/README.md:8]: change version

File: roles/oneview_hypervisor_manager/meta/main.yml [hunk 4]
```diff
@@ -0,0 +1,16 @@
+---
+galaxy_info:
+  role_name: oneview_fc_network
```
Comment (VenkateshRavula) [inline roles/oneview_hypervisor_manager/meta/main.yml:3]: change role name and also description

**Review by VenkateshRavula — CHANGES_REQUESTED** [review]
roles for oneview_hypervisor_manager_facts is missing

**Comment by nabhajit-ray** [comment]
> roles for oneview_hypervisor_manager_facts is missing

added

File: roles/oneview_hypervisor_manager/meta/main.yml (hunk 4 above)
Comment (nabhajit-ray) [reply roles/oneview_hypervisor_manager/meta/main.yml]: done

File: roles/oneview_hypervisor_manager/README.md (hunk 3 above)
Comment (nabhajit-ray) [reply roles/oneview_hypervisor_manager/README.md]: done

**Comment by nabhajit-ray** [comment]
> roles for oneview_hypervisor_manager_facts is missing

Added

File: roles/oneview_hypervisor_manager_facts/README.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+
+## Example Playbook
+
+```yaml
+- hosts: all
+  collections:
+    - name: hpe.oneview
+  roles:
+    - hpe.oneview.oneview_hypervisor_manager
```
Comment (VenkateshRavula) [inline roles/oneview_hypervisor_manager_facts/README.md:27]: change role name

**Review by VenkateshRavula — APPROVED** [review]

**Pull request merged**